.github/workflows/CI.yml

```diff
@@ -27,7 +27,7 @@ jobs:
 
       - name: Run build
         run: |
-          ./gradlew build -Dopensearch.version=1.3.0-SNAPSHOT
+          ./gradlew build -Dopensearch.version=2.1.0-SNAPSHOT
```

Comment:
Is it stable version? I wonder cause for knn we switched to 2.0.0-alpha1-snapshot, latest announcement was that alpha1 works with lucene 9.1

Reply:
We will be releasing this along with 2.1 ( not during 2.0).  The expectation is that they will fix issues in 2.x ( 2.1 ) first and down-port to 2.0.